commands/create/nodepool/command.go

```diff
@@ -51,7 +51,7 @@ follows:
 - Availability zones: the node pool will use 1 zone selected randomly.
 - Instance type (AWS) / VM Size (Azure): the default machine type of the installation will be
   used. Check 'gsctl info' to find out what that is.
-- Scaling settings: on AWS, the minimum will be 3 and maximum 10 nodes, and on Azure, the node count will be 3
+- Scaling settings: on AWS, the minimum will be 3 and the maximum 10 nodes, and on Azure, the minimum will be 3 and the maximum 3 nodes.
```

Comment:
We just discussed these defaults in SIG product and think that thexy should be the same on AWS and Azure (min=3, max=10). See https://github.com/giantswarm/roadmap/issues/251

Reply:
agreed, let's make it the same for both providers.